src/centos/7/rpmpkg/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7
+
+# Install the rpm packaging toolchain we need to build rpms
+RUN yum install -y \
+        gcc \
+        make \
+        rpm-build \
+        ruby-devel \
+        rubygems \
+        sudo \
```

Comment:
Should this be doing a `yum clean all` at the end?

Reply:
I'm not sure - I've used this file as model: https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/master/src/centos/7/mlnet/Dockerfile

@dagood you had specified "yum clean" commands in the suggested solution - issue https://github.com/dotnet/dotnet-buildtools-prereqs-docker/issues/92. Are those commands needed?

Reply:
It's a guideline, but yes, the `yum` cache data is not useful for the image user and should be cleaned up in the same `RUN` statement as the install commands. The user either won't run any more `yum` commands and have no use of a cache, but even if they do install more packages, they will probably find the cache data is old and useless.